docs/compression_algorithms/Pruning.md

```diff
@@ -69,6 +69,18 @@ and reduce the corresponding accuracy drop even before model training. This opti
 sparsity and filter pruning algorithms. It can be enabled by setting a non-zero value of `num_bn_adaptation_samples` in the
 `batchnorm_adaptation` section of the `initializer` configuration (see example below).
 
+#### Interlayer ranking types
+
+
+Interlayer ranking type can be one of `unweighted_ranking` or `learned_ranking`.  
+ In case of `unweighted_ranking` and with  
+ `all_weights=True` all filter norms will be collected together and sorted to choose less important ones. But this approach may not be optimal because filter norms are a good measure of filter importance inside a layer, but not across layers.   
```

Comment:
Some information about how this method is called ("Learned Global Ranking") and a reference to the paper is required.

Reply:
Added link to the paper and algo name.